mkosi.images/system/mkosi.conf

```diff
@@ -15,8 +15,10 @@ ExtraTrees=
 
 PostInstallationScripts=mkosi.sanitizers.chroot
 
+Environment=
+        SYSTEMD_REPART_OVERRIDE_FSTYPE_ROOT=%F
```

Comment:
Probably an ignorant question, but why is this needed if the default filesystem is the default?

Reply:
Because the default filesystem only applies when mkosi generates its own repart partition definition files. This bit is to make it apply with the custom configuration we ship in mkosi.repart/